code/game/objects/structures/mannequin.dm

```diff
@@ -0,0 +1,1187 @@
+			clothing[SLOT_MANNEQUIN_FEET] = clothToWear
+		else
+			qdel(clothToWear)
+	if(mapping_gloves)
+		var/obj/item/clothToWear = new mapping_gloves(src)
+		if(canEquip(null, SLOT_MANNEQUIN_GLOVES, clothToWear))
+			clothing[SLOT_MANNEQUIN_GLOVES] = clothToWear
+		else
+			qdel(clothToWear)
+	if(mapping_ears)
+		var/obj/item/clothToWear = new mapping_ears(src)
+		if(canEquip(null, SLOT_MANNEQUIN_EARS, clothToWear))
+			clothing[SLOT_MANNEQUIN_EARS] = clothToWear
+		else
+			qdel(clothToWear)
+	if(mapping_suit)
+		var/obj/item/clothToWear = new mapping_suit(src)
+		if(canEquip(null, SLOT_MANNEQUIN_OCLOTHING, clothToWear))
+			clothing[SLOT_MANNEQUIN_OCLOTHING] = clothToWear
+		else
+			qdel(clothToWear)
+	if(mapping_glasses)
+		var/obj/item/clothToWear = new mapping_glasses(src)
+		if(canEquip(null, SLOT_MANNEQUIN_EYES, clothToWear))
+			clothing[SLOT_MANNEQUIN_EYES] = clothToWear
+		else
+			qdel(clothToWear)
+	if(mapping_belt)
+		var/obj/item/clothToWear = new mapping_belt(src)
+		if(canEquip(null, SLOT_MANNEQUIN_BELT, clothToWear))
+			clothing[SLOT_MANNEQUIN_BELT] = clothToWear
+		else
+			qdel(clothToWear)
+	if(mapping_hat)
+		var/obj/item/clothToWear = new mapping_hat(src)
+		if(canEquip(null, SLOT_MANNEQUIN_HEAD, clothToWear))
+			clothing[SLOT_MANNEQUIN_HEAD] = clothToWear
+		else
+			qdel(clothToWear)
+	if(mapping_mask)
+		var/obj/item/clothToWear = new mapping_mask(src)
+		if(canEquip(null, SLOT_MANNEQUIN_MASK, clothToWear))
+			clothing[SLOT_MANNEQUIN_MASK] = clothToWear
+		else
+			qdel(clothToWear)
+	if(mapping_back)
+		var/obj/item/clothToWear = new mapping_back(src)
+		if(canEquip(null, SLOT_MANNEQUIN_BACK, clothToWear))
+			clothing[SLOT_MANNEQUIN_BACK] = clothToWear
+		else
+			qdel(clothToWear)
+	if(mapping_id)
+		var/obj/item/clothToWear = new mapping_id(src)
+		if(canEquip(null, SLOT_MANNEQUIN_ID, clothToWear))
+			clothing[SLOT_MANNEQUIN_ID] = clothToWear
+		else
+			qdel(clothToWear)
+
+	if(mapping_hand_right)
+		var/obj/item/clothToWear = new mapping_hand_right(src)
+		held_items[GRASP_RIGHT_HAND] = clothToWear
+	if(mapping_hand_left)
+		var/obj/item/clothToWear = new mapping_hand_left(src)
+		held_items[GRASP_LEFT_HAND] = clothToWear
+
+	update_icon()
+
+
+/obj/structure/mannequin/proc/spin()
+	change_dir(turn(dir, 90))
+
+/obj/structure/mannequin/verb/rotate_mannequin()
+	set name = "Rotate Mannequin"
+	set category = "Object"
+	set src in oview(1)
+
+	if(!usr || !isturf(usr.loc))
+		return
+
+	if(anchored)
+		return
+
+	if(usr.isUnconscious() || usr.restrained())
+		return
+
+	spin()
+
+
+/obj/structure/mannequin/fat
+	name = "fat human marble mannequin"
+	icon_state="mannequin_marble_fat_human"
+	fat = 1
+
+/obj/structure/mannequin/vox
+	name = "vox marble mannequin"
+	icon_state="mannequin_marble_vox"
+	species_type = /datum/species/vox
+
+/obj/structure/mannequin/monkey
+	name = "monkey marble mannequin"
+	icon_state="mannequin_marble_monkey"
+	primitive = 1
+	clothing_offset_y = -5*PIXEL_MULTIPLIER
```

Comment:
They don't. Monkeys are sprited in the center of the tile so items in their hands (for instance) appear without any offset, just as if they were in human hands.